libs/sdk-ui-dashboard/src/presentation/drill/DrillConfigPanel/DrillToUrl/ParameterDetails/AttributeDisplayFormParameterDetail.tsx

```diff
@@ -0,0 +1,123 @@
+// (C) 2020-2022 GoodData Corporation
+import React, { useState, useEffect } from "react";
+import { IntlShape, useIntl } from "react-intl";
+import { LRUCache } from "@gooddata/util";
+import { IAttributeElement, ObjRef, objRefToString } from "@gooddata/sdk-model";
+import { ParameterDetail } from "./ParameterDetail";
+import { AttributeDisplayFormType } from "../../../types";
+import { useBackendStrict } from "@gooddata/sdk-ui";
+import { IAnalyticalBackend, IElementsQueryResult } from "@gooddata/sdk-backend-spi";
+
+const MAX_CACHED_REQUESTS = 50;
+const MAX_URL_LENGTH = 100;
+const DISPLAY_FORM_ELEMENTS_LIMIT = 3;
+
+const requestCache = new LRUCache<IElementsQueryResult>({ maxSize: MAX_CACHED_REQUESTS });
+
+const getDisplayFormLabel = (type: string | undefined) => {
+    switch (type) {
+        case AttributeDisplayFormType.HYPERLINK:
+            return "configurationPanel.drillIntoUrl.editor.urlDisplayFormTypeLabel";
```

Comment:
Please use defineMessage or defineMessages from react-intl so that the i18n validation can detect this.